chainer/training/updaters/multiprocess_parallel_updater.py

```diff
@@ -133,11 +133,13 @@ def __init__(self, iterators, optimizer, converter=convert.concat_examples,
             # The context is not initialized, it will be fine.
             _cuda_initialized = False
         if _cuda_initialized:
-            raise RuntimeError(
-                'The CUDA context has been already initialized. '
-                'MultiprocessParallelUpdater assumes the context is '
-                'uninitialized. Please do not call CUDA API before '
-                'MultiprocessParallelUpdater creates processes.')
+            if multiprocessing.get_start_method() != 'spawn':
```

Comment:
- Check `'forkserver'`.  (Thanks @kmaehashi)
- `if _cuda_initialized and multiprocessing.get_start_method() != 'spawn':`?

Reply:
Update on this.
`forkserver` also works.

There is a problem when using `spawn` and `forkserver`
the `converter` attribute of the `MultiprocessParallelUpdater` can't be pickled.